Add introduction to Neo4j Workspace (#300)
neo4j/docs-getting-started: modules/ROOT/pages/index.adoc

link:https://neo4j.com/product/neo4j-graph-database/?ref=product/[Neo4j Graph Database^] is the core product, a native graph database that is built to store and retrieve connected data. + 
link:https://neo4j.com/licensing/[Two editions^], a Community Edition and an Enterprise Edition, are available. +
There are various ways to get started with Neo4j: 

* link:https://neo4j.com/docs/operations-manual/current/installation/[by installing on-premises] 
* link:https://neo4j.com/docs/operations-manual/current/cloud-deployments/[by deploying on the cloud platform] of your choice
* by using {aura_signup}[Neo4j Aura]

You can find all necessary documentation on how to install and maintain Neo4j in the link:https://neo4j.com/docs/operations-manual/current/[Operations manual^]. 

[[neo4j-aura]]
== Neo4j Aura

> *Neo4j Aura* is a fully managed cloud service offering, that includes *AuraDB* and *AuraDS*.

Reviewer: ```suggestion
*Neo4j Aura* is a fully managed cloud service offering.
Two versions are available, *AuraDB* and *AuraDS*.
```

Author: I'm not sure that it's correct to call AuraDB and AuraDS two versions of Aura. It may confuse users.
